config/migration/migration_types/types.go

```diff
@@ -0,0 +1,16 @@
+package migration_types
+
+type Migratable interface {
+	Save(filename string) error
+	ToNext() (Migratable, error)
+}
+
+type NullConfig struct{}
```

Comment:
empty

Reply:
?

Reply:
isn't it already empty? not sure if I understood